cassandra-mesos-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/framework/Main.java

```diff
@@ -134,7 +136,8 @@ private static int _main() throws UnknownHostException {
         final long      healthCheckIntervalSec      = Long.parseLong(       Env.option("CASSANDRA_HEALTH_CHECK_INTERVAL_SECONDS").or("60"));
         final long      bootstrapGraceTimeSec       = Long.parseLong(       Env.option("CASSANDRA_BOOTSTRAP_GRACE_TIME_SECONDS").or("120"));
         final String    cassandraVersion            =                       "2.1.4";
-        final String    frameworkName               = frameworkName(        Env.option("CASSANDRA_CLUSTER_NAME"));
```

Comment:
The option for cluster is an existing parameter, however it used to be overloaded to configure framework name (by prepending `cassandra.`) and then that framework name would be used as the cluster name.

This change makes it so that `CASSANDRA_CLUSTER_NAME` can be provided to specify the cluster name (any string the user wants, if nothing is specified a default value of `cassandra` is used), the framework name will then prepend `cassandra.` to be consistent with the existing framework naming scheme.